installers/win-installer/project.nsh

```diff
@@ -1,15 +1,15 @@
 
 ; Projects file generated by release.gradle
-!define PROJECT_BUILD_DIR "C:\Projects\pcgen\pcgen-clean\pcgen\build"
-!define PROJECT_DIST_BIN_DIR "C:\Projects\pcgen\pcgen-clean\pcgen\build\nsis"
-!define PROJECT_DIST_DIR "C:\Projects\pcgen\pcgen-clean\pcgen\build"
+!define PROJECT_BUILD_DIR "/home/javier/NetBeansProjects/pcgen/build"
```

Comment:
This is setting off a red flag. Is there a way to do this in a more generic way? Its not blocking though.